docs/user-guide/configuration.md

```diff
@@ -3,31 +3,32 @@ id: configuration
 title: Configuration
 ---
 
-Search `.htmlhintrc` file in current directory and all parent directories:
+By default, htmlhint looks for a `.htmlhintrc` file in the current directory
+and all parent directories, and applies its rules when parsing a file:
```

Comment:
Please remove the line break
We can use `Toggle Word Wrap` in editors like vscode if we want to edit markdown files

Reply:
I added it on purpose, as reasoned [here](https://rhodesmill.org/brandon/2012/one-sentence-per-line/). In particular, short lines allow more readable diffs, works better with CLI pagers, etc. But if you insist, I can remove it.

Reply:
Nice explanation 👍 
I will let decide @thedaviddias what style we want to use in the future

Just my personal experience in the past was that people have problems with markdown when using GFM vs Markdown native

And specially this case should not break in a new line
Otherwise we could rewrite it into a list of bullet-points 🤔

Reply:
@waldyrious I agree with @Shinigami92 in that point, I believe it's better to keep it in one line. I'm not too afraid about diffs in that particular case.

Reply:
Sure @thedaviddias, no problem — I've removed the line break. I do invite you to read the post linked above ([Semantic Linefeeds](https://rhodesmill.org/brandon/2012/one-sentence-per-line/)) regardless; it's quite interesting.